kubernetes-response-engine/playbooks/README.md

```diff
@@ -0,0 +1,164 @@
+# Playbooks
```

Comment:
I was initially confused, thinking that this was related to ansible. You might want to ask around to see if others had the same confusion.

Reply:
Yes, you are not the unique who pointed us to this misunderstanding. 

A few days ago, @KnoxAnderson suggested to use this name because is well known among security teams.

https://docs.microsoft.com/en-us/azure/security-center/security-center-playbooks
https://blogs.cisco.com/security/using-a-playbook-model-to-organize-your-information-security-monitoring-strategy

So we chose the term "Playbook", perhaps a possible alternative would be "Security Playbooks"?